### PR TITLE
Notick dp2 homepage search fix

### PR DESCRIPTION
**Pull request by nadinequinr3**
- Primarily, this is a fix to ensure searches are filtered to the current version a search is performed from. Previously, searches from the root page of some versions were not filtered correctly, resulting in no results returned on https://docs.r3.com/en/platform/corda/1.5/cenm.html or https://docs.r3.com/en/platform/corda/5.0-dev-preview-2.html, for example. After a lot of investigation, I could not figure out why the version is not picked up for these pages so I'm applying this fix to ensure searches work as expected. Currently there is a bad user experience when directed to https://docs.r3.com/en/platform/corda/5.0-dev-preview-2.html but searches don't work. When we know more about hugo menus, etc, we could try implement a cleaner fix.

- Also made some minor fixes in relation to DP2 — broken link, updated name/title for landing page, release notes, and checksums.

### CODE REVIEW

File: themes/doks/layouts/partials/head/script-header.html [hunk 1]
```diff
@@ -13,17 +13,34 @@
...
+            {{ $project := .Page.Params.project}}
+            {{ $version := .Page.Params.version}}
+            window.facetFilters = [
+              ['project:{{- $project -}}'],
+              ['version:{{- $version -}}'],
+              'language:en'
+            ];
+            {{ else }}
+              {{ $project := .Parent.Page.Params.project}}
+              {{ $version := .Parent.Page.Params.version}}
```
Comment (yarivus) [inline themes/doks/layouts/partials/head/script-header.html:36]: You might have some indentation issues in the changes of this file.

**Review by yarivus — COMMENTED** [review]
checkpointing my review ;)
I left a minor comment on indent.

**Review by yarivus — APPROVED** [review]
LGTM

**Review by colin-hillis — APPROVED** [review]
LGTM

**Pull request merged**